ui/src/maps/csrfToken.ts

```diff
@@ -0,0 +1,11 @@
+import { CsrfToken } from "@/models/CsrfToken"
+import { CsrfTokenResponse } from "@/types/csrfTokenResponse"
+
+export const mapCsrfTokenResponseToCsrfToken = (response: CsrfTokenResponse): CsrfToken => {
+    return {
+        token: response.token,
+        expiration: new Date(response.expiration),
+        issued: new Date(),
+    }
+}
+
```

Comment:
This'll bring this in line with other mappers

```suggestion
import { CsrfToken } from "@/models/CsrfToken"
import { CsrfTokenResponse } from "@/types/csrfTokenResponse"
import { MapFunction } from '@/services/mapper'

export const mapCsrfTokenResponseToCsrfToken: MapFunction<CsrfTokenResponse, CsrfToken> = function(source) {
    return {
        token: source.token,
        expiration: this.map('string', source.expiration, 'Date'),
        issued: new Date(),
    }
}

```